benchmarkapps/BenchmarkServer/Hubs/EchoHub.cs

```diff
@@ -44,5 +44,10 @@ public void SendPayload(string payload)
         {
             // Dump the payload, we don't care
         }
+
+        public DateTime NTP()
+        {
+            return DateTime.UtcNow;
```

Comment:
As I mentioned in the other PR, let's call this something like "GetCurrentTime" rather than "NTP" to avoid confusion.